build.sh

```diff
@@ -630,8 +621,10 @@ case "$COMMAND" in
         VERSION=$(sh build.sh get-version)
         cd ..
 
-        mkdir -p ${TEMPDIR}/realm-cocoa-${VERSION}/osx
+        mkdir -p ${TEMPDIR}/realm-cocoa-${VERSION}/osx/swift
+        mkdir -p ${TEMPDIR}/realm-cocoa-${VERSION}/ios/static
         mkdir -p ${TEMPDIR}/realm-cocoa-${VERSION}/ios/dynamic
+        mkdir -p ${TEMPDIR}/realm-cocoa-${VERSION}/ios/swift
         mkdir -p ${TEMPDIR}/realm-cocoa-${VERSION}/browser
         mkdir -p ${TEMPDIR}/realm-cocoa-${VERSION}/Swift
```

Comment:
It's where we store `RLMSupport.swift`. See the release package structure in the opening comment of this PR.